pkgs/development/python-modules/drms/default.nix

```diff
@@ -44,16 +44,19 @@ buildPythonPackage rec {
     pytest-doctestplus
   ];
 
-  disabledTests = [ "test_query_hexadecimal_strings" ];
+  disabledTests = [
+    "test_query_hexadecimal_strings"
```

Comment:
Why did you disable this test? It has passed on hydra.
https://hydra.nixos.org/build/275370481/nixlog/1

Reply:
Oh, sorry, I saw wrong. 🤦